app/src/main/res/values/strings_questionnaire.xml

```diff
@@ -11,13 +11,96 @@
         <item>60~</item>
         <item>secret</item>
     </string-array>
-    <string-array name="questionnaire_values_age">
-        <item>10代</item>
-        <item>20代</item>
-        <item>30代</item>
-        <item>40代</item>
-        <item>50代</item>
-        <item>60代</item>
-        <item>答えたくない</item>
+    <string name="questionnaire_android_user">How long have you been using Android?</string>
+    <string-array name="questionnaire_keys_android_user">
+        <item>None</item>
+        <item>less than a half year</item>
+        <item>0.5~1 year</item>
+        <item>1~2 years</item>
+        <item>2~3 years</item>
+        <item>3~4 years</item>
+        <item>4~5 years</item>
+        <item>more than 5 years</item>
     </string-array>
+    <string name="questionnaire_android_developer">How long have you been developing Android app?</string>
+    <string-array name="questionnaire_keys_android_developer">
+        <item>None</item>
+        <item>less than a half year</item>
+        <item>0.5~1 year</item>
+        <item>1~2 years</item>
+        <item>2~3 years</item>
+        <item>3~4 years</item>
+        <item>4~5 years</item>
+        <item>more than 5 years</item>
+    </string-array>
+    <string name="questionnaire_job_category">Please tell me the job category?</string>
+    <string-array name="questionnaire_keys_job_category">
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_job_category appears to be unused [UnusedResources]